automation/automation.sh

```diff
@@ -46,6 +46,12 @@ else
         cycle_infra)
             /opt/venv/bin/python /haitaton-gis/process_data.py ylre_katualueet $tormays_source
             ;;
+        tram_infra)
+            /opt/venv/bin/python /haitaton-gis/process_data.py ylre_katualueet $tormays_source
+            ;;
+        tram_lines)
+            /opt/venv/bin/python /haitaton-gis/process_data.py ylre_katualueet $tormays_source
```

Comment:
If I start the script with `cycle_infra tram_infra tram_lines`, does this mean that the `ylre_katualueet` processing is done separately three times, once for each target?

Reply:
Yes it does. I will make own ticket where this will be solved out so that it will be processed only once.

Reply:
Here is the ticket:
https://helsinkisolutionoffice.atlassian.net/browse/HAI-2597